src/article.js

```diff
@@ -36,7 +30,7 @@ class Article extends Component {
 }
 
 //list the props
-Article.PropTypes = {
+Article.propTypes = {
```

Comment:
When I was showing everyone how to use propTypes I realised that the reason it never worked was that I got the capitalisation wrong for propTypes. I still struggle with spelling related bugs as well!